src/types.ts

```diff
@@ -98,3 +100,10 @@ export enum NullAddresses {
 }
 
 export const EthCoinIndex = '60';
+
+export type NormalizedSource = { 
+  url?: string,
```

Comment:
@JohnnyJumper Why do we need this overloaded type with all optional keys? Lesser optional keys is better - please follow the proposed structure https://github.com/unstoppabledomains/resolution/pull/115/files#